src/main/java/org/terasology/hunger/HungerAuthoritySystem.java

```diff
@@ -112,10 +114,15 @@ public void postBegin() {
     @ReceiveEvent
     public void onPeriodicActionTriggered(PeriodicActionTriggeredEvent event, EntityRef entityUnused) {
         if (event.getActionId().equals(HUNGER_DAMAGE_ACTION_ID)) {
-            for (EntityRef entity : entityManager.getEntitiesWith(HungerComponent.class, AliveCharacterComponent.class)) {
+            for (EntityRef entity : entityManager.getEntitiesWith(HungerComponent.class,
+                    AliveCharacterComponent.class)) {
                 HungerComponent hunger = entity.getComponent(HungerComponent.class);
-                hunger.lastCalculatedFood = Math.max(0,
-                        hunger.lastCalculatedFood - (healthDecreaseInterval * hunger.foodDecayPerSecond) / 1000);
+                expectedDecay = (healthDecreaseInterval * hunger.foodDecayPerSecond) / 1000;
+                // Send event to allow for other systems to modify hunger decay.
+                AffectHungerEvent affectHungerEvent = new AffectHungerEvent(expectedDecay);
+                entity.send(affectHungerEvent);
+                expectedDecay = affectHungerEvent.getResultValue();
+                hunger.lastCalculatedFood = Math.max(0, hunger.lastCalculatedFood - expectedDecay);
```

Comment:
I don't think we need to change the value of the "expected decay". We use it to trigger `AffectHungerEvent` and afterwards are only interested in the result.

```suggestion
                hunger.lastCalculatedFood = Math.max(0, hunger.lastCalculatedFood - affectHungerEvent.getResultValue());
```